jadx-gui/src/main/java/jadx/gui/settings/JadxSettings.java

```diff
@@ -65,6 +67,10 @@
 
 	private int settingsVersion = 0;
 
+	@JadxSettingsAdapter.GsonExclude
+	@Parameter(names = { "-sc", "--select-class" }, description = "GUI: Open the selected class and show the decompiled code")
+	private String cmdSelectClass = null;
```

Comment:
You don't need to add this field, it is already inherited from `JadxCLIArgs`

Reply:
I think this is a misunderstanding. This option can not be inherited from `JadxCLIArgs` because it does not exist in jadx-cli as it has no GUI to select a class.

I don't want to misuse one of the existing command-line arguments as this option is GUI only.

Reply:
Hm, sorry, I thought you will use `--single-class` option (easier way), but you add a new one just for jadx-gui. And now for me unclear what `--single-class` will mean for jadx-gui :)

Reply:
I have not checked it but from my perspective there are other command-line options like `--export-gradle` or `--no-src` that should not have an effect in jadx-gui. 

To make it clean there would be still the possibility to split `JadxCLIArgs` into two classes, one base class for argument that affect settings settings (and as base class for `JadxSettings`) and a second class that inherits base class and contains the cli-only arguments.

Reply:
Yeah, you are right, these must be different classes.
I just too lazy and don't want to write more code and more classes :)

Reply:
Hm, I have merged PR, but just notice that `jadx-gui --help` don't show new option :(

Reply:
I managed to fix this issue. See PR #660